hw/ip/otbn/dv/model/otbn_trace_checker.h

```diff
@@ -75,6 +75,11 @@ class OtbnTraceChecker : public OtbnTraceListener {
   // through MatchPair if there is any.
   const OtbnIssTraceEntry::IssData *PopIssData();
 
+  // Tell the model to not execute checks to see if secure wiping has written random data
+  // to all registers before wiping them with zeroes if set.
+  bool no_sec_wipe_data_chk;
```

Comment:
Can you move this flag to the `private` section below? And I've used a convention in this class where members get trailing underscores: could you follow that too?